### PR TITLE
Updated texts

### PR DESCRIPTION
**Pull request by jancurn**

### CODE REVIEW

File: src/apify_helpers.js [hunk 1]
```diff
@@ -198,39 +198,41 @@ const getActorAdditionalFields = async (z, bundle) => {
...
             key: 'timeoutSecs',
             required: false,
             default: timeoutSecs || 0,
             type: 'integer',
         },
         {
+            // TODO: This should be a drop-down with allowed memory sizes, using values from apify-shared/contants
             label: 'Memory',
-            helpText: 'Amount of memory allocated for the actor run, in megabytes.',
+            helpText: 'Amount of memory allocated for the actor run, in megabytes. The more memory, the faster will your actor run.',
```
Comment (davidjohnbarton) [inline src/apify_helpers.js]: The more memory, the faster your actor will run.

File: src/searches/fetch_items.js [hunk 2]
```diff
@@ -59,25 +59,27 @@ module.exports = {
...
             {
                 label: 'Limit',
+                helpText: 'The maximum number of dataset items to fetch. If empty, the default limit will be used.',
                 key: 'limit',
                 required: false,
                 type: 'integer',
             },
             {
                 label: 'Offset',
+                helpText: 'The offset in the dataset where to start fetching the items from. If empty, it will be from the beginning.',
```
Comment (davidjohnbarton) [inline src/searches/fetch_items.js]: The offset in the dataset from where to start fetching the items.

File: src/triggers/actor_additional_fields.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     noun: 'Actor Additional Fields',
     display: {
         label: 'Actor Additional Fields',
-        description: 'This is a hidden trigger, and is used to test getActorAdditionalFields function.',
+        description: 'This is a hidden trigger, which is used to test getActorAdditionalFields function.',
```
Comment (davidjohnbarton) [inline src/triggers/actor_additional_fields.js]: This is a hidden trigger used to test the getActorAdditionalFields function.

File: src/triggers/actors.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ module.exports = {
     noun: 'Actors',
     display: {
         label: 'List of actors',
-        description: 'This is a hidden trigger, and is used in a Dynamic Dropdown within this app',
+        description: 'This is a hidden trigger, which is used in a Dynamic Dropdown within this app',
```
Comment (davidjohnbarton) [inline src/triggers/actors.js]: This is a hidden trigger used in a Dynamic Dropdown within this app

File: src/triggers/tasks.js [hunk 5]
```diff
@@ -24,8 +24,7 @@ module.exports = {
     noun: 'Tasks',
     display: {
         label: 'List of Tasks',
-        // TODO: Is this correct message?
-        description: 'This is a hidden trigger, and is used in a Dynamic Dropdown within this app',
+        description: 'This is a hidden trigger, which is used in a Dynamic Dropdown within this app',
```
Comment (davidjohnbarton) [inline src/triggers/tasks.js]: This is a hidden trigger used in a Dynamic Dropdown within this app

**Comment by jancurn** [comment]
Fixed, ready to merge

**Pull request merged**